### DISCUSSION TITLE
Character Counter

### DISCUSSION THREAD

**Discussion by insertish** [stoatchat/discussions#27] [Feature Suggestions]
Display current character count in the bottom left of editor / message box.

**Comment by sussycatgirl** [comment]
Maybe make it only appear if you get close to the character limit

**Reply by Kokoduckiee** [reply]
Yeah, I agree that it needs some indication near limit

**Comment by TehAngryXeno** [comment]
I 2nd this!

**Comment by gabrielfordevelopment** [comment]
https://github.com/stoatchat/for-web/pull/1131

**Comment by Dadadah** [comment]
This has been [merged](https://github.com/stoatchat/for-web/pull/1131), and will be present in the [next version](https://github.com/stoatchat/for-web/pull/1116) of for-web.

**Comment by infi** [comment]
Implemented.
